rskj-core/src/test/java/co/rsk/peg/BridgeSerializationUtilsTest.java

```diff
@@ -1009,6 +1012,27 @@ public void deserializeInteger() {
         Assert.assertEquals(1200, BridgeSerializationUtils.deserializeInteger(RLP.encodeBigInteger(BigInteger.valueOf(1200))).intValue());
     }
 
+    @Test
+    public void serializeScript() {
+        Script expectedScript = ScriptBuilder.createP2SHOutputScript(2, Lists.newArrayList(new BtcECKey(), new BtcECKey(), new BtcECKey()));
```

Comment:
Maybe it is a good time to add some direct explicit test exercising `ScriptBuilder`

Reply:
@ajlopezrsk  not sure I got it correctly. Do you mean adding some tests for `ScriptBuilder` class? It's from the third-party library.